qt/python/mantidqt/widgets/instrumentview/presenter.py

```diff
@@ -12,6 +12,7 @@
 """
 from mantidqt.widgets.observers.ads_observer import WorkspaceDisplayADSObserver
 from mantidqt.widgets.observers.observing_presenter import ObservingPresenter
+from workbench.config import get_window_config
```

Comment:
I'm afraid this is quite the right way around. The `mantidqt` package can't depend on `workbench` as it's a collection of standalone widgets that can and are used in other applications, hence the parent being passed through in the code on master so the caller of the presenter determines what the parent is. It may well be used on its own for example with no parent at all.

Reply:
The same is true for all of the other windows. I think we will need to add the flags to be passed through the presenter.

Reply:
Ah yes, my bad I had completely forgot that these widgets had to be independent from workbench, so unfortunately didn't raise it sooner. I guess this will miss out this realise, but at least we'll have time to sort the other things out such as the interfaces.

Reply:
Yes this is definiltely worth pushing back now, paricularly as testing has already started and this affects a lot of things. I'll put it in 6.1S1